test/e2e/singlecluster/e2e_test.go

```diff
@@ -686,6 +687,68 @@ var _ = ginkgo.Describe("Kueue", func() {
 			}, util.LongTimeout, util.Interval).Should(gomega.BeTrue())
 		})
 
+		ginkgo.It("Should run with prebuilt workload", func() {
+			var wl *kueue.Workload
+			ginkgo.By("Create the pebuilt workload and the job adopting it", func() {
+				sampleJob = (&testingjob.JobWrapper{Job: *sampleJob}).
+					Label(constants.PrebuiltWorkloadLabel, "prebuilt-wl").
+					Image("gcr.io/k8s-staging-perf-tests/sleep:v0.0.3", []string{"5s", "-termination-grace-period", "0s"}).
+					BackoffLimit(0).
```

Comment:
Please also add terminationGracePeriod 1, for the case when the pods start running, but the SIGTERM trap is not registered yet. Otherwise the termination will take 30s

Reply:
done